src/components/EnvironmentOverride/environmentOverride.scss

```diff
@@ -1,15 +1,34 @@
 .environment-override {
-    padding: 12px 24px;
+    max-width: 800px;
+    height: 100%;
+    padding: 20px 24px;
+    margin: 0 auto;
+
     section:not(:first-child) {
         margin-top: 24px;
     }
+
+    .form__title {
+        font-size: 16px;
+        font-weight: 600;
+    }
+
+    .form__title,
+    .form__subtitle {
+        color: var(--N900);
+    }
+
     .form__label {
         &.bold {
             margin-top: 0;
             margin-bottom: 16px;
             font-size: 14px;
         }
     }
+
+    .rotateBy-180 {
```

Comment:
can we use dorumale class instead of this?

Reply:
We don't have the formula as of now. There's one class though in base.scss named `.rotate` but it uses the var from inline/global CSS (wherever available).

We'll have to introduce the new formula for transform-rotate. As of now, I'm keeping this as is & we can work on introducing formulae for different transform properties.